cmd/osv-scanner/main.go

```diff
@@ -265,6 +202,22 @@ func run(args []string, stdout, stderr io.Writer) int {
 	return 0
 }
 
+func getAllCommands(commands []*cli.Command) []string {
```

Comment:
any reason not returning a map? since we are checking if it contains something later

Reply:
Yeah, map does give better performance for checking. But, since we only need to get all valid commands here (which is a very small amount), I think it's fine to use a slice as it offers better readability.